src/xrpld/app/tx/detail/Transactor.cpp

```diff
@@ -794,35 +801,36 @@ Transactor::checkSingleSign(
     else if (sleAccount->isFieldPresent(sfRegularKey))
     {
         // Signing key does not match master or regular key.
-        JLOG(j.trace()) << "checkSingleSign: Not authorized to use account.";
+        JLOG(ctx.j.trace())
+            << "checkSingleSign: Not authorized to use account.";
         return tefBAD_AUTH;
     }
     else
     {
         // No regular key on account and signing key does not match master key.
         // FIXME: Why differentiate this case from tefBAD_AUTH?
-        JLOG(j.trace()) << "checkSingleSign: Not authorized to use account.";
+        JLOG(ctx.j.trace())
+            << "checkSingleSign: Not authorized to use account.";
         return tefBAD_AUTH_MASTER;
     }
 
     return tesSUCCESS;
 }
 
+// TODO generalize
```

Comment:
seems to be generalized enough already ?

Reply:
> seems to be generalized enough already ?

If there was anything more I wanted to do with it, I don't remember what it was. 😁